jupyter/utils/install_pdf_deps.sh

```diff
@@ -2,6 +2,8 @@
 
 # Install dependencies required for Notebooks PDF exports
 
+set -e
```

Comment:
Yep, that's how this is normally done

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

There are some piped commands below, so also consider

> # set -o pipefail
> 
> The bash shell normally only looks at the exit code of the last command of a pipeline. This behavior is not ideal as it causes the -e option to only be able to act on the exit code of a pipeline’s last command. This is where `-o pipefail` comes in.

For better build log, I also suggest `set -x`